Cargo.toml

```diff
@@ -19,30 +19,31 @@ tokio = { version = "1.35", features = ["sync"], default-features = false }
 tokio-stream = { version = "0.1", default-features = false }
 tracing = { version = "0.1", default-features = false }
 thiserror = "1"
-wasm-bindgen-futures = { version = "0.4.41", optional = true }
-wasm-tokio-retry = { package = "tokio-retry", git = "https://github.com/polytope-labs/rust-tokio-retry", branch = "master", optional = true }
-native-tokio-retry = { package = "tokio-retry", version = "0.3.0", optional = true }
+getrandom = { version = "0.2.12", default-features = false }
+again = { git = "https://github.com/softprops/again", branch = "develop", default-features = false }
 
 # optional dependencies
 subxt = { version = "0.34", optional = true }
+wasm-bindgen-futures = { version = "0.4.41", optional = true }
 
 [features]
-default = []
+default = ["native"]
 native = [
     "futures/std",
     "serde_json/std",
     "tokio/rt-multi-thread",
     "tracing/std",
     "jsonrpsee/ws-client",
-    "native-tokio-retry",
+    "again/default",
 ]
 subxt = ["dep:subxt"]
 web = [
     "jsonrpsee/jsonrpsee-types",
     "jsonrpsee/async-wasm-client",
     "jsonrpsee/client-web-transport",
     "wasm-bindgen-futures",
-    "wasm-tokio-retry"
+    "again/wasm-bindgen",
+    "getrandom/js",
```

Comment:
they also didn't add js features to `getrandom` luckily, cargo allows us to do this